routes/socket.js

```diff
@@ -23,6 +25,8 @@ const handleSocketConnection = async (io, socket) => {
 
   logger.info('Client connected successfully', { token, roomId });
 
+  io.to(`${gameMasterSocketId}`).emit('player:joined', { player: socket.id });
```

Comment:
```suggestion
  io.to(gameMasterSocketId).emit('player:joined', { player: socket.id });
```

Kan vi få tilbake username her også? Eg trenger det for å presentere det i lobbyen

Reply:
Nå nei, vi har ingen brukerdatabase enda

Reply:
men jeg kan legge ved en placeholder for nå